clients/fidesui/src/hoc/CustomTypography.tsx

```diff
@@ -65,11 +65,14 @@ const CustomParagraph = ({
 
 const CustomLink = React.forwardRef<
   HTMLAnchorElement,
-  React.ComponentProps<typeof Typography.Link> & CustomTypographyTextProps
->(({ size, ...props }, ref) => (
+  React.ComponentProps<typeof Typography.Link> &
+    CustomTypographyTextProps & { primaryColor?: boolean }
+>(({ size, primaryColor, ...props }, ref) => (
```

Comment:
Maybe not a show-stopper (feel free to move on without), but it would be ideal to pass "primary" as the property value rather than a boolean here for scalability. Something like `variant="primary"`.